canonical-websites/www.ubuntu.com
# Demo of wider grid and bump in font-size for bigger screens

## Done
This is a demo accompanying the draft proposal we're working on with @clagom and  @spencerbygraves for unifying nav behaviour and look, increasing the fixed-width and bumping root font-size for bigger screens across all Canonical sites.

Please do not merge or review as this is only meant to support discussion with stakeholders.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Look for anything that doesn't look quite right due to the wider site width, particularly changes in text wrapping.


## Issue / Card

[List of links to Github issues/bugs and cards if needed - e.g. `Fixes #1`]

## Screenshots

[if relevant, include a screenshot]
